packages/rlc-common/src/static/pollingContent.ts

```diff
@@ -61,9 +61,7 @@ export async function getLongRunningPoller<TResult extends HttpResponse>(
   };
 
   {{#if useLegacyLro}}
-  const poller = new LroEngine(poller, options);
-  await poller.poll();
```

Comment:
We don't need this if it's useLegacyLro now ? I remember previously the poller instance won't call the inital request if we don't add poller.poll() right after creating it.

Reply:
As you mentioned earlier, changing it to call the `poll` method will cause a breaking change, because the helper will be async and will need to be awaited. The `useLegacyLro` flag should be used to not introduce breaking changes but for the majority of the cases, introducing the breaking change is fine because most RLC libraries are still in beta.

Reply:
Two follow-up questions 

- Is the latest version `2.5.0` backward compatible with lower version like `2.2.0`? Do we need to bind the version to older one in package.json when the `useLegacyLro` enabled
- Why we remove the code line `await poller.poll();` the `useLegacyLro` enabled here?